8Knot/pages/contributions/visualizations/pr_review_response.py

```diff
@@ -0,0 +1,275 @@
+from dash import html, dcc, callback
+import dash
+import dash_bootstrap_components as dbc
+from dash.dependencies import Input, Output, State
+import plotly.graph_objects as go
+import pandas as pd
+import logging
+from dateutil.relativedelta import *  # type: ignore
+import plotly.express as px
+from pages.utils.graph_utils import get_graph_time_values, color_seq
+from queries.pr_response_query import pr_response_query as prr
+import io
+from cache_manager.cache_manager import CacheManager as cm
+from pages.utils.job_utils import nodata_graph
+import time
+import app
+
+PAGE = "contributions"
+VIZ_ID = "pr-review-response"
+
+gc_pr_review_response = dbc.Card(
+    [
+        dbc.CardBody(
+            [
+                html.H3(
+                    "Pull Request Review Responses",
```

Comment:
I think the title and the info need a bit more detail.

A title idea is: "PR Conversation Activity Policy"

and an info idea is: "Tracks the number of PRs that are open on a given day vs. those that have received a comment or a review within a time interval, or that are waiting on a response from the opener."

Reply:
This is rough but I'm trying to help describe the difference between the other visualization that you created and this one.

Reply:
Maybe "PR Conversation Response Time" or "PR Lifecycle Response Time" ?

Reply:
"PR Conversation Engagement"?

Reply:
That works for me ill update it now